pyproject.toml

```diff
@@ -215,4 +216,6 @@ exclude = [
   "instrumentation-genai/opentelemetry-instrumentation-langchain/examples/**/*.py",
   "instrumentation-genai/opentelemetry-instrumentation-weaviate/tests/**/*.py",
   "instrumentation-genai/opentelemetry-instrumentation-weaviate/examples/**/*.py",
+  "instrumentation-genai/opentelemetry-instrumentation-mcp/tests/**/*.py",
+  "instrumentation-genai/opentelemetry-instrumentation-mcp/examples/**/*.py",
```

Comment:
I'm disappointed this exclude section grew so much and by new projects... It was not supposed to be used by new projects.

I don't think it makes sense to add those 2. Supporting proper type hints from the beginning is easier.